### PR TITLE
Fix: Restructure to align with rest of RFCs

### PR DESCRIPTION
**Pull request by georgepadayatti**

### CODE REVIEW

**Review by lalc — APPROVED** [review]
LGTM. Happy to move to v1.0 approved version.

File: ewc-rfc004-individual-wallet-attestation.md [hunk 1]
```diff
@@ -3,17 +3,15 @@
...
 **Reviewers:**
-* Dr Andreas Abraham (ValidatedID, Spain)
-* Mr George Padaytti (iGrant.io, Sweden)
 * Mr Lal Chandran (iGrant.io, Sweden)
+* Dr Andreas Abraham (ValidatedID, Spain)
 * Dr Nikos Triantafyllou (University of the Aegean, Greece)
 
 **Status:** Work in progress
 
-Current: OAuth 2.0 Attestation-Based Client Authentication Draft 03 [[1](#60reference)]
```
Comment (lalc) [inline ewc-rfc004-individual-wallet-attestation.md:15]: Do we need this? or can we remove?

File: ewc-rfc004-individual-wallet-attestation.md [hunk 2]
```diff
@@ -22,77 +20,84 @@ Current: OAuth 2.0 Attestation-Based Client Authentication Draft 03 [[1](#60refe
...
 
 # 3.0	Wallet Unit Attestation (WUA)
 
-The attestation of a wallet's capabilities through Wallet Unit Attestation is based on the OAuth 2.0 Attestation-Based Client Authentication draft 03[1].
+The attestation of a wallet's capabilities through Wallet Unit Attestation is based on the OAuth 2.0 Attestation-Based Client Authentication draft 03 [3].
 
 ## 3.1 Wallet Unit Attestation JWT
 
-The following Wallet Unit Attestation JWT must be presented when sending the token request as defined in RFC001[7].
+The following Wallet Unit Attestation JWT must be presented when sending the token request defined in RFC001[8] to issuers or when sending the authorisation response as described in RFC002 [9] to relying parties.
```
Comment (ed-curran) [inline ewc-rfc004-individual-wallet-attestation.md:46]: should we include an example of a presentation authorization response? or do you feel these two RFCs combined already have enough info.

Comment (lalc) [reply ewc-rfc004-individual-wallet-attestation.md:46]: Good idea. Do you want to suggest an example?

Comment (ed-curran) [reply ewc-rfc004-individual-wallet-attestation.md:46]: I would prefer to add presentation authorization response in a separate PR. Can we keep this one as is just to token request for now? For authorization response may need to consider some extra things.

**Review by andreasabr — APPROVED** [review]
Looks good, great work

File: ewc-rfc004-individual-wallet-attestation.md (hunk 2 above)
Comment (ed-curran) [reply ewc-rfc004-individual-wallet-attestation.md:46]: if this addition reference presentation authorisation response  is removed i would approve this PR, and we can deal with presentation authorisation response in a different PR

Comment (georgepadayatti) [reply ewc-rfc004-individual-wallet-attestation.md:46]: 1. So, should we merge this for now and include the example for the authorisation response as a separate addition? 
2. The current approach should suffice for the direct post-response mode in my opinion. Do you agree ?
3. Is your concern about the fragment mode?

**Comment by lalc** [comment]
@georgepadayatti Pls update version number in title before merging.

File: ewc-rfc004-individual-wallet-attestation.md (hunk 2 above)
Comment (ed-curran) [reply ewc-rfc004-individual-wallet-attestation.md:46]: well i would remove the addition of "or when sending the authorisation response as described in RFC002 [9] to relying parties" and add that together with examples in another PR.

I don't have any specific thoughts about authorization response i just feel like this restructuring PR should just be restructuring and adding  authorization response would be an addition.

**Review by endimion — APPROVED** [review]

File: ewc-rfc004-individual-wallet-attestation.md (hunk 1 above)
Comment (georgepadayatti) [reply ewc-rfc004-individual-wallet-attestation.md:15]: Removed

**Pull request merged**